frontend/src/components/new-position-form.js

```diff
@@ -0,0 +1,151 @@
+import React from "react";
+import { moment } from "moment";
+
+const toNames = ({ first_name, last_name }) => `${first_name} ${last_name}`;
+
+const validNumber = val => (val && isNaN(val) ? "Not a valid number" : false);
+const validDate = val =>
+    (val && moment(val, "YYYY-MM-DD", true).isValid()) || val === ""
+        ? false
+        : "Not a valid date";
```

Comment:
What?? `validDate === false` means that the date is valid? Don't mix variables that store booleans and strings. If it is supposed to be an error message, make it an error message.

Reply:
Now empty str is returned representing no err msg